packages/rpc-types/src/__tests__/coercions-test.ts

```diff
@@ -60,10 +60,10 @@ describe('coercions', () => {
             expect(coerced).toBe(raw);
         });
         it('throws on an out-of-range `UnixTimestampUnsafeBeyond2Pow53Minus1`', () => {
-            const thisThrows = () => unixTimestamp(BigInt(8.75e15));
+            const thisThrows = () => unixTimestamp(BigInt(2e63));
```

Comment:
This doesn't actually do what it says on the box (also it's not `2**63` but rather `2 * 10**63`)

```ts
BigInt(2e63); // 2000000000000000115715919885453939654786757378350080876345294848n
new Intl.NumberFormat().format('2E63'); // "2,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000"
```

I think what you're looking for is this.


```suggestion
            const thisThrows = () => unixTimestamp(BigInt('0xffffffffffffffff') + 1n);
```

Reply:
Wait I had these changed to `2 ** 63`, maybe I missed this one? Or I fat-fingered a rebase.

Reply:
That still won't work.

```ts
2 ** 63; // 9223372036854776000
2n ** 63n; // 9223372036854775808n
BigInt('0x8000000000000000'); // 9223372036854775808n
```

Reply:
Ohh I see now, thanks.